lib/driver.rb

```diff
@@ -43,5 +44,67 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def get_total_revenue
+      total_rev = 0.0
+      @trips.each do |trip|
+        if trip.cost != nil
+          trip_cost = (trip.cost - 1.65) * 0.8
```

Comment:
Magic numbers! https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Pull these out into variables or constants:

```ruby
RIDE_SHARE_FEE = 1.65
DRIVER_SHARE_PERCENTAGE = 0.80

def get_total_revenue
  # ...
  trip_cost = (trip.cost - RIDE_SHARE_FEE) * DRIVER_SHARE_PERCENTAGE
end
```